projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4993,7 +4996,7 @@ public function alert_auto_ssl_fail() {
 					$this.html( <?php echo wp_json_encode( __( 'Checking', 'jetpack' ) ); ?> );
 					$( '#jetpack-recheck-ssl-output' ).html( '' );
 					e.preventDefault();
-					var data = { action: 'jetpack-recheck-ssl', 'ajax-nonce': '<?php echo $ajax_nonce; ?>' };
+					var data = { action: 'jetpack-recheck-ssl', 'ajax-nonce': '<?php echo $ajax_nonce; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- do we want to escape nonces ?>' };
```

Comment:
Nonces should be safe, but on the other hand it's easy enough to do `wp_json_encode()` like we did in the other PR.
```suggestion
 					var data = { action: 'jetpack-recheck-ssl', 'ajax-nonce': <?php echo wp_json_encode( $ajax_nonce ); ?> };
```